src/webportal/deploy/webportal.yaml.template

```diff
@@ -91,7 +91,24 @@ spec:
         - name: WEBPORTAL_PLUGINS
           # A raw JSON formatted value is required here.
           value: |
-            {{ cluster_cfg['webportal']['plugins'] }}
+            [
+{% if cluster_cfg['cluster']['common']['marketplace'] == "true" %}
```

Comment:
What if:

```
1. The admin deploys marketplace in a previous version of PAI
2. He upgrades PAI to `v1.6.0`
```

Will there be two duplicate marketplace entries here?

Reply:
Yes, if the old webportal-plugin still in services-configurations.yaml, there will be two duplicate marketplace entries here.

Reply:
I will add an optional config to disable the auto-added marketplace-plugin in webportal.
The admin can avoid the duplicate marketplace entries by it.